src/Simulation/TargetDefinitions/Interfaces/IType4Core.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
Cassandra typically lets me know that the part `All rights reserved` is out-of-date. IIRC, based on [this](https://docs.opensource.microsoft.com/releasing/maintain/copyright-headers/).

Reply:
I'll at least update that in the files I'm changing, but avoid changing otherwise unmodified files for now.